### DISCUSSION TITLE
Consiglio per sensore allarme

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#3132] [Hardware & Firmware]
Sto realizzando un allarme con sensoristica Zigbee (sensori di movimento e porta). Ho la necessità di installare due sensori all'interno di altrettante intercapedini dato che queste hanno una griglia per mezzo della casa si può di fatto entrare in casa. Avevo pensato di installare due sensori di movimento ma il problema è che dalle griglie, soprattutto una delle due, in alcune ore del pomeriggio entra molta luce solare (soprattutto in questo periodo) con il risultato che il sensore scatta continuamente ingannato dalla luce. Ho quindi pensato di rimuovere questi sensori e di installare dei sensori di vibrazione alle due finestre a vasistas che dalle intercapedini permettono di entrare in casa. Non ho usato sensori porta in quanto queste due finestre soprattutto d'estate sono sempre aperte pertanto non voglio essere obbligato a chiuderle. Trattandosi di finestre a vasistas anche se tenute aperte non permettono il passaggio di una persona se non si provvede a forzarle per aprirle ulteriormente. Il problema è che il sensore di vibrazione, sebbene permetta di settare la sensibilità, di fatto ha bisogno che gli venga impressa una certa accelerazione perchè scatti. Anche alla massima sensibilità. Voglio dire che se uno muove lentamente la finestra non scatta.

Qualcuno a qualche consiglio da darmi. Non esiste un sensore di vibrazione non basato sull'accelerazione impressa ma effettivamente sul fatto che lo stesso venga mosso seppur lentamente?

**Comment by Menhir83** [comment]
Magari ci starebbe bene un bel motorino per vasistas in associazione ad un sensore di contatto, in questo modo puoi decidere apertura/chiusura delle 2 finestre ed avere comunque il sistema armato con la conferma di tutti i varchi chiusi. I motori esistono di diverse dimensioni con forze diverse. Non sono sicuro che siano smart ma si possono adattare facilmente. Quindi, nel momento che attivi l´allarme, si chiudono le 2 finestre in automatico e poi il sistema viene armato (per esempio.. non so nemmeno quanto possono essere veloci ma credo qualche decina di secondi). Anche se penso che il sistema si possa armare considerando lo stato del sensore al momento dell´attivazione, quindi se quelle due finestre sono aperte, l´allarme si attiva lo stesso.

**Comment by 50494554524F** [comment]
ci sono sensori di movimento radar

**Comment by array81** [comment]
Intanto grazie delle risposte.
In merito all'apertura/chiusura automatica delle finestra basculanti questo mi complicherebbe di molto la cosa oltre a necessitare dell'elettricità in loco.
Il problema elettricità lo avrei anche per i sensori di presenza a onde millimetriche purtroppo. Sinceramente pensavo che oltre a sensori di vibrazione esistesse qualcosa a (batteria) che rilevasse il movimento di un oggetto. I sensori di vibrazione lo fanno ma c'è bisogno di un impulso, se uno (il ladro) muove lentamente la finestra il sensore non rileva nulla.

**Comment by Fantalucio** [comment]
Ciao @array81,
perché non posizioni il sensore in basso in modo che resti chiuso anche quando sono posizionate a vasistas?

**Reply by array81** [reply]
Grazie. Sinceramente non ci avevo pensato ma funziona, la vasistas ruota poco quando è aperta e queste fa si che installando il sensore ai lati nella parte bassa il movimento sia talmente limitato che il sensore resta comunque chiuso anche se apro la finestra. Al tempo stesso se a vasistas aperta un ladro dovesse infilare una mano per sganciare i perni nella parte superiore e aprire la finestra questo farebbe scattare il sensore sensore.
